modules/releasing/pages/adjusting-timeouts-resources.adoc

```diff
@@ -0,0 +1,77 @@
+= Adjusting timeouts and resources
+
+When a Release Pipeline is executed through a PipelineRun, the Pipeline will run until a timeout is reached. This timeout is usually an hour and will make the Release Pipeline to fail if it doesn't finish on time.
+
+To overcome this, there are two strategies {ProductName} users can follow.
+
+== Adjust timeouts
+
+Whenever a pipeline is defined through the tenantPipeline/finalPipeline fields in the ReleasePlan or the pipeline field in the ReleasePlanAdmission, users can specify a `timeouts` field in the following way:
+
+[source,yaml]
+----
+pipeline:
+  pipelineRef:
+    resolver: git
+    params:
+      - name: url
+        value: "<url-to-repo>.git"
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: "<path-to-your-pipeline>"
+  timeouts:
+    pipeline: "2h0m0s" <.>
+    tasks: "1h0m0s" <.>
+    finally: "1h0m0s" <.>
+----
+
+<.> specifies the timeout for the entire PipelineRun. Defaults to the global configurable default timeout of 60 minutes.
```

Comment:
I think since this is adding documentation to this, it would be good to mention the constraint here that will cause tekton webhooks to reject the PLR if not followed
I *think* that is pipeline timeout *must* = tasks + finally, but it may just be pipeline timeout *must* >= tasks+finally

Reply:
Added an admonition on this.